pmg/static/resources/javascript/pmg.js

```diff
@@ -168,4 +168,10 @@ $(function() {
       "share", "width=600, height=400, scrollbars=no");
       ga('send', 'social', 'linkedin', 'share', url);
   });
-});
+
+  $(".page-share.email").on("click", function(e) {
+    e.preventDefault();
+    window.location = 'mailto:?subject=' + document.title + '&body=I saw this on pmg.org.za and thought it might interest you: ' + url;
```

Comment:
Use `encodeURIComponent(document.title)` and similarly for the body, otherwise a ? in the document title or url will break things.